store/src/java/com/zimbra/cs/service/authenticator/ClientCertAuthenticator.java

```diff
@@ -120,7 +128,7 @@ private void captureClientCert(X509Certificate cert) {
             // Write in text form
             wr = new OutputStreamWriter(os, Charset.forName("UTF-8"));
             wr.write("-----BEGIN CERTIFICATE-----\n");
-            wr.write(new sun.misc.BASE64Encoder().encode(buf));
+            wr.write(Base64.getEncoder().encodeToString(buf));
```

Comment:
Has this been tested?

Reply:
Yes I did write a test program and tested that the files created are same. Also this is code is executed only when a DebugConfig is true. By default it is false.